main/python/cmdLineUtils.py

```diff
@@ -1171,33 +1179,49 @@ def _rootLsPrintSimpleLs(keyList,indent,oneColumn):
         if (i+1)%ncol != 0 and i != len(keyList)-1:
             if not IS_TERMINAL: write( \
                 key.GetName().ljust(col_widths[i%ncol]))
-            elif isDirectoryKey(keyList[i]): write( \
+            elif isDirectoryKey(key):
+                write( \
```

Comment:
Let's try to get rid of these `\` if you already change the line of code. Here we can maybe combine it with the next line? Or the next two lines? If it's max. 120 chars that would be okay.

Reply:
Any tool you suggest for passing it through an autoformatter? autopep8? black?